server/controllers/profile.js

```diff
@@ -32,9 +24,20 @@ exports.loadProfile = asyncHandler(async (req, res, next) => {
     throw new Error("Not authorized");
   }
 
-  res.status(200).json({
-    success: {
-      profile: profile,
-    },
-  });
+  res.status(200).json({ success: { profile } });
 });
+
+// @route GET /profile/listings
+// @desc load users from db
+// @access Public
+exports.populateListings = async (req, res) => {
+  const { numOfUsers } = req.query;
```

Comment:
Good work! Additionally, you can have pagination with page numbers/ profiles per page :)

A video I found on the internet: https://www.youtube.com/watch?v=soWg_UtD_AM